### DISCUSSION TITLE
Feature request: Selling "waves" of tickets

### DISCUSSION THREAD

**Discussion by saschafoerster** [pretix/pretix#1965] [Ideas]
We are missing the possibility the sell tickets in "waves": 

I searched the documentation and found to use cases: 
https://docs.pretix.eu/en/latest/user/events/structureguide.html#use-case-early-bird-tiers-based-on-dates
https://docs.pretix.eu/en/latest/user/events/structureguide.html#use-case-early-bird-tiers-based-on-ticket-numbers

"Waves" would be a mixed use-case. You have a certain amount of tickets, that you sell, until they are sold out. Then you have to join the waiting list. After some time the next "wave" of tickets can be bought. And waiting list again. Maybe there is a third or more waves following. 

We had two problems with this kind of ticket-selling: 
- The waiting list was a new waiting list for every product, so we couldn't take people from the first waves waiting list into the third wave, if they were to slow for the second one. We only had people from the second waiting list be notified, when the third wave started. 
- Vouchers, that should be available during the whole time, weren't available after our first product (for the first wave) was sold out. We manually had to switch the voucher to be available in product nr. 2 (wave 2), after product 1 (wave 1) was sold out. 

So how could this be solved? Maybe it could be possible to just create 1 product but then to have some automation or trigger date for contingents. So the contingents will be added on a certain date.

**Comment by raphaelm** [comment]
> Vouchers, that should be available during the whole time, weren't available after our first product (for the first wave) was sold out. We manually had to switch the voucher to be available in product nr. 2 (wave 2), after product 1 (wave 1) was sold out.

This is solvable by assigning the voucher to a special quota instead of a product.

> The waiting list was a new waiting list for every product, so we couldn't take people from the first waves waiting list into the third wave, if they were to slow for the second one. We only had people from the second waiting list be notified, when the third wave started.

Yes, this is a problem we need to fix in some way.

**Comment by saschafoerster** [comment]
> This is solvable by assigning the voucher to a special quota instead of a product.

Ah, I indeed oversaw it before! <img width="1081" alt="Gutschein" src="https://user-images.githubusercontent.com/1153741/64493733-351b2c00-d284-11e9-9458-670c5db5140b.png">
